examples/qdrant-vectorstore-example/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/tmc/langchaingo/examples/qdrant-vectorstore-example
+
+go 1.21
+
+require github.com/tmc/langchaingo v0.1.3
```

Comment:
while developing this I'd recommend that you point to current main via `go get github.com/tmc/langchaingo@main`